cmd/access/node_builder/access_node_builder.go

```diff
@@ -575,6 +577,15 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 		AdminCommand("read-execution-data", func(config *cmd.NodeConfig) commands.AdminCommand {
 			return stateSyncCommands.NewReadExecutionDataCommand(builder.ExecutionDataStore)
 		}).
+		Module("transactions and collections storage", func(node *cmd.NodeConfig) error {
+			// TODO: needs to be wrapped with ChainedCollections module, otherwise once we switch
```

Comment:
Link the issue as TODO here https://github.com/onflow/flow-go/issues/6523#issuecomment-2722425528 . 

Will be addressed separately. We can review and approve this PR, but not merge until the TODO is completed. 

cc @fxamacker